### ISSUE TITLE
Use new DataONE format Ids

### ISSUE THREAD

**Issue by gothub** [NCEAS/recordr#67]
When uploading R scripts to DataONE, ensure that the new CCI format type
is used: 

```
<objectFormat>
<formatId>application/R</formatId> 
<formatName>R script file format</formatName>
<formatType>DATA</formatType>
<mediaType name="text/plain" />
<extension>R</extension>
</objectFormat>
```

This will be available in CCI 2.3.1 as mentioned [here](https://redmine.dataone.org/issues/7960).
I don't think that any of the other new format ids apply for recordr (Python, Matlab, SAS).

This is also a note to verify that DataONE is registering this file type correctly.

**Comment by mbjones** [comment]
I think they settled on a different media type for the format, FYI.  You
might check the most recent issue thread to see where it stands.

**Comment by gothub** [comment]
@mbjones yes, thanks - further down on the issue page it indicates that the 
format that was implemented is actually:

```
<objectFormat>
<formatId>application/R</formatId> 
<formatName>R programming language script</formatName>
<formatType>DATA</formatType>
<mediaType name="text/x-rsrc" />
<extension>R</extension>
</objectFormat>
```
